src/PrestaShopBundle/Form/Admin/Sell/Product/DataTransformer/TypeaheadRedirectionTargetTransformer.php

```diff
@@ -30,6 +30,12 @@
 
 use Symfony\Component\Form\DataTransformerInterface;
 
+/**
+ * @deprecated since 1.7.9.0
+ *
+ * This transformer was useful when the form used a TypeAhead form type, with the new EntitySearchInputType
+ * it became useless because the format is more adapted by default and not complex enough to justify a transformer.
+ */
 class TypeaheadRedirectionTargetTransformer implements DataTransformerInterface
```

Comment:
Please deprecate using coding standards : https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations

Reply:
And if available, deprecate the service